### PR TITLE
redo query to improve performance

### PR DESCRIPTION
**Pull request by malaffoon**
The optimizer just wasn't dealing with this query. In dev, it went from 1.5h to 12s with this change.

### CODE REVIEW

File: task-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -73,11 +73,9 @@ task:
 #    cron: 0 0 4 * * *
 
 taskSendReconciliationReport:
+  # default root settings
   query: status=PROCESSED&after=-PT24H
   log: false
-
-#taskSendReconciliationReport:
-#  # there is no root (non-tenant) configuration for this task
```
Comment (malaffoon) [inline task-service/src/main/resources/application.yml]: unrelated, clean up annotated configuration file

File: migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml [hunk 2]
```diff
@@ -1228,28 +1228,30 @@ sql:
...
-              FROM exam_target_score wets
-                JOIN exam we ON wets.exam_id = we.id
-                JOIN target wt ON wt.id = wets.target_id
+                sub.id,
+                sub.exam_id,
+                sub.target_id,
+                sub.student_relative_residual_score,
+                sub.standard_met_relative_residual_score,
+                :migrate_id AS migrate_id
+              FROM (SELECT wets.id, wets.exam_id, wets.target_id, wets.student_relative_residual_score, wets.standard_met_relative_residual_score
```
Comment (esotrope) [inline migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: surprised this is faster since there is a sub select.

Comment (malaffoon) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: The optimizer completely borked it for some reason. It was joining the exam-target-score table with the three reference tables FIRST, getting 350m results, then joining the exam table. This just forces it to do it the other way around.

Comment (agorina) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: Just thinking loud: would it make sense to make the same change for the reporting migrate?

Comment (malaffoon) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: Probably. But no problems have been reported and i've heard that i don't cut enough corners ...

**Pull request merged**